src/App.jsx

```diff
@@ -1,33 +1,55 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+export const App = () => {
+  const getFilterMovies = (movies, query) => {
+    let prepearedMovies = [...moviesFromServer];
+    const lowerQuery = query.toLowerCase().trim();
+
+    prepearedMovies = prepearedMovies.filter(
+      movie => movie.title.toLowerCase().includes(lowerQuery)
+        || movie.description.toLowerCase().includes(lowerQuery),
+    );
+
+    return prepearedMovies;
+  };
```

Comment:
we can just use movies directly without the redundant copy
```suggestion
  const getFilterMovies = (movies, query) => {
    const lowerQuery = query.toLowerCase().trim();

    return movies.filter(
      movie => movie.title.toLowerCase().includes(lowerQuery)
        || movie.description.toLowerCase().includes(lowerQuery),
    );
  };
```